### PR TITLE
[SITE-988] WordPress Composer GA release note

### PR DESCRIPTION
**Pull request by stevector**
Adds [Release Note](https://pr-9248-documentation.appa.pantheon.site/release-notes/2024/10/wordpress-composer-managed-ga) for WordPress (Composer Managed) ~general availability~ graduation from Early Access.

### CODE REVIEW

**Comment by IngridKwok** [comment]
@stevector @rachelwhitton  Made some changes. Please review to see if it's good to go.

**Comment by rachelwhitton** [comment]
@jazzsequence I know you have an additional commit planned for this one, I was thinking we should also mention Bedrock in both the title and in the summary when discussing why this is beneficial

File: source/releasenotes/2024-10-16-wordpress-composer-managed-ga.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: WordPress (Composer Managed) is now in general availability
+published_date: "2024-10-16"
+categories: [new-feature, wordpress, documentation]
+---
+
+We are excited to announce that our [WordPress (Composer Managed)](https://github.com/pantheon-systems/wordpress-composer-managed) upstream is now in general availability. This means it is fully supported in production environments.
```
Comment (stevector) [inline source/releasenotes/2024-10-16-wordpress-composer-managed-ga.md:7]:
```suggestion
We are excited to announce that our [WordPress (Composer Managed)](https://github.com/pantheon-systems/wordpress-composer-managed) upstream is now in general availability. This means it is fully supported in production environments.

Historically we have provided multiple (sometimes conflicting) examples of how to use Composer with WordPress on Pantheon. This upstream unifies many of the best practices we have learned while leveraging [Roots/Bedrock](https://roots.io/bedrock/).
```

**Comment by stevector** [comment]
I think we should mention bedrock and I added a suggestion for it

File: source/releasenotes/2024-10-16-wordpress-composer-managed-ga.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ categories: [new-feature, wordpress, documentation]
 
 We are excited to announce that our [WordPress (Composer Managed)](https://github.com/pantheon-systems/wordpress-composer-managed) upstream is now in general availability. This means it is fully supported in production environments.
 
+Historically we have provided multiple (sometimes conflicting) examples of how to use Composer with WordPress on Pantheon. This upstream unifies many of the best practices we have learned while leveraging [Roots/Bedrock](https://roots.io/bedrock/). Composer allows more fine-grained control over your site's dependencies (plugins, themes, and even WordPress core), places the power of who determines what components should be updated and when into the hands of the development teams that support it and simplifies your git repositories. We're excited to unlock this enhanced workflow for our WordPress customers.
```
Comment (stevector) [inline source/releasenotes/2024-10-16-wordpress-composer-managed-ga.md:9]:
```suggestion
Historically we have provided multiple (sometimes conflicting) examples of how to use Composer with WordPress on Pantheon. This upstream unifies many of the best practices we have learned while leveraging [Roots/Bedrock](https://roots.io/bedrock/). Composer allows more fine-grained control over your site's dependencies (plugins, themes, and even WordPress core). Composer also places the power of who determines what components should be updated and when into the hands of the development teams that support it and simplifies your Git repositories. We're excited to unlock this enhanced workflow for our WordPress teams.
```

**Comment by IngridKwok** [comment]
I’ve completed another round of review on the additions.

**Comment by jazzsequence** [comment]
Refactor PR to

* focus on docs updates
* note that WPCM support is "minimal support" in some way
* it is no longer EA

"WPCM is no longer in EA. It’s now available as an open source upstream available for our customers and the community at large."

**Comment by jazzsequence** [comment]
@stevector @scottbuscemi 
I rephrased the release note based on current input from product re: EA/GA/other to remove reference to General Availability and instead say "it is no longer Early Access". Most of the rest of the content can (I think) stay.

**Comment by jazzsequence** [comment]
Closing as WPCM is not yet in GA

**Pull request closed without merging**